README.md

```diff
@@ -24,25 +23,55 @@ The simplest way to obtain the code is using the github .zip feature.
+1. Download and install BYOND, login into it
+1. Download and install Git and VSC
+1. Create a folder in which you wish to keep the repository files, preferably on an SSD, then right click → "Git Bash Here". A new command prompt should appear.
+1. Inside the fork you have created above, click the "Code" ribbon and copy the URL under the HTTPS section
+1. Run the command `git clone URL_FROM_ABOVE`, changing the "URL_FROM_ABOVE" text with the URL you have copied before, and let it run to completion
+1. Open VSC and select "Open Folder", select the newly created folder
+1. Accept the prompts, install the extensions that are suggested, login to those who asks you to, reboot VSC as requested/needed
 
-    git clone https://github.com/Aurorastation/Aurora.3.git
+You are now ready to develop for/using the Aurorastation codebase.
 
-(hint: hold down ctrl and press insert to paste into git bash)
+If you intend to submit changes to us, please **create a new branch, do not edit the master branch**, you can do so in VSC by bringing up the command palette with CTRL+SHIFT+P and run the command "Git: Create Branch From", select the master branch, give a name for the new one and then hit enter.
 
-This will take a while to download, but it provides an easier method for updating.
+You can then perform changes, commit to said branch, publish it and create a Pull Request (PR) to our repository, to have it evaluated and eventually included in our master branch.
+
+To update your master branch when ours is updated, simply go to your fork, click the "Sync fork" ribbon and then "Update Branch".
+
+You can always [join us on Discord](https://discord.gg/0sYA49zHYGnKWM9p) for questions.
 
 ### INSTALLATION
 
-First-time installation should be fairly straightforward.  First, you'll need BYOND installed.  You can get it from [here](http://www.byond.com/).
+This is a sourcecode-only release, so the next step is to compile the server files.
+
+**Using BYOND tools natively is not supported, you need to either:**
+
+- Compile the server into a runnable binary using the build tools documented [here](tools/build/README.md).
+
+- Follow the instructions in the **Development** section, you can skip the creation of a Github account and the use of Git if you intend to manually update your server, but it's suggested not to.
 
-This is a sourcecode-only release, so the next step is to compile the server files.  Open aurorastation.dme by double-clicking it, open the Build menu, and click compile.  This'll take a little while, and if everything's done right you'll get a message like this:
+  Once you are inside VSC, with the codebase folder opened and all the extensions installed, simply hit CTRL+F5 (Run without debugging) to compile the source code, this might take a little, after which you should see DreamMaker and/or DreamSeeker popping up, and then the full client UI.
```

Comment:
```suggestion
Once you are inside VSC, with the codebase folder opened and all the extensions installed, simply hit CTRL+F5 (Run without debugging) to compile the source code. This might take a little, after which you should see DreamMaker and/or DreamSeeker popping up, and then the full client UI.
```